cmd/tracee-ebpf/main.go

```diff
@@ -26,6 +30,8 @@ var traceeInstallPath string
 var listenMetrics bool
 var metricsAddr string
 var enrich bool
+var logStdout = logger.Default()
```

Comment:
> In my opinion we don't need log for both stdout and stderr.

It would demand a way to set multi writers for an unique logger. I think it can be done using `zapcore.NewMultiWriteSyncer`. 

What if the user require a unique logger with a single writer? It seems the way that tracee is doing this [currently](https://github.com/aquasecurity/tracee/blob/main/cmd/tracee-ebpf/main.go#L65-L66).

> The logger configuration and levels should indicate to where the log should go.

I agree with you that this is a well tied approach. But as I mentioned, if tracee or other project requires to log to different outputs and/or to use different levels for that matter, it would preferable to use a loose approach, letting the user to set it on its own.

Reply:
I think we're already overly flexible and non standard in how we log. 
IMO this PR should be used to solve that problem, not add more flexibility. I think Alon's approach would be better for that.
We also need to remove the debug drilling we used throughout the code, as it's clutter in a lot of cases.
Calling logger.Debug(...) should be enough and should output when running tracee in debug mode.

Reply:
> We also need to remove the debug drilling we used throughout the code, as it's clutter in a lot of cases. Calling logger.Debug(...) should be enough and should output when running tracee in debug mode.

I think this PR is already removing the drilling if the package importer decide to use the package level functions. But for sure we can remove the possibility to create new user-defined loggers letting all importers relying on a singleton.